src/Pos/Wallet/Web/Doc.hs

```diff
@@ -310,6 +311,15 @@ instance ToSample CWalletRedeem where
             , crSeed     = "1354644684681"
             }
 
+instance ToSample CPostVendWalletRedeem where
+    toSamples Proxy = singleSample sample
+      where
+        sample = CPostVendWalletRedeem
+            { pvWalletId         = CAddress $ CHash "1fSCHaQhy6L7Rfjn9xR2Y5H7ZKkzKLMXKYLyZvwWVffQwkQ"
+            , pvSeed             = "1354644684681"
+            , pvBackupPhrase     = mkBackupPhrase12 ["garlic", "swim", "arrow", "globe", "note", "gossip", "cabin", "wheel", "sibling", "cigar", "person", "clap"]
```

Comment:
I think, just `backupPhrase` (at cost of duplicates in docs) would be better, but it's imo & minor